simulariumio/converter.py

```diff
@@ -222,6 +222,10 @@ def _check_agent_ids_are_unique_per_frame(self) -> bool:
                 get_n_subpoints = True
         return True
 
+    @staticmethod
+    def _format_timestep(number: float):
+        return float("%.4g" % number)
```

Comment:
Is 4 decimal places enough precision for the time step?

Reply:
I think so, we're only displaying 3 sig figs in the time bar currently, I've never seen a simulation timestep with more than 2 sig fig precision

Reply:
this is the trouble with passing through a text file format first.  we have to round things off before it even gets to the player. in a perfect world everything would be a literal float until it's time to display something and THEN it gets rounded.

Reply:
Following the arraybuffer update, that will be an option. Whether or not we choose to implement this, it would be wise to support the text version despite this particular limitation.